src/main/java/org/maxgamer/QuickShop/Shop/ShopManager.java

```diff
@@ -402,8 +402,8 @@ public void handleChat(final Player p, String msg) {
                             }
                         }
                         signBlockState.setType(Material.OAK_WALL_SIGN);
-                        final Sign sign = (Sign) signBlockState.getData();
-                        sign.setFacingDirection(bf);
+                        org.bukkit.block.data.type.WallSign signBlockDataType = (org.bukkit.block.data.type.WallSign) signBlockState.getBlockData();
```

Comment:
Since you imported WallSign, can you adjust this to use the short name, for code readability?
```suggestion
                        WallSign signBlockDataType = (WallSign) signBlockState.getBlockData();
```